City-of-Helsinki/drupal-asuntotuotanto
# Max-width: 100% fix

### Description

Some elements goes over the screen


### How to test

- make fresh
- check that frontpage and meistä pages are working fine and elements are not going over on screen
